lib/widgets/content.dart

```diff
@@ -620,23 +619,6 @@ class UserMention extends StatelessWidget {
 //     borderRadius: BorderRadius.all(Radius.circular(3))));
 }
 
-class MessageUnicodeEmoji extends StatelessWidget {
```

Comment:
Let's squash deleting this into the same commit that makes it no longer needed.

Meanwhile, from the commit message:

    This widget displayed the emoji name as a fallback in a bordered
    container. But now that we rely on TextSpan to display the emoji
    codepoints text, we can't reliably determine the fallback condition
    _for now_.

I don't understand the last bit here. What fallback condition are you thinking of; and what would need to change between "for now" and being able to determine it?

Reply:
This commits talks about the possibility to determine if the emojiUnicode is present in a supported emoji list. If not we could fallback to showing this container. See [discussion](https://github.com/zulip/zulip-flutter/pull/245#discussion_r1300891469).

Reply:
nvm, see [comment](https://github.com/zulip/zulip-flutter/pull/245#discussion_r1301020631)